config/locales/pl.yml

```diff
@@ -404,6 +404,10 @@ pl:
       teacher_school_email_warning: >-
         Czy to na pewno Twój szkolny adres-email? Jeżeli tak, wciśnij %{button} poniżej.
         Jeżeli nie, wpisz adres e-mail przyznany Ci przez Twoją szkołę.
+      did_you_mean:
+        TODO Did you mean
```

Comment:
Can we get the correct translations in here?

Reply:
Done. Please review the updated text.